src/devices/Bmxx80/samples/Bme680.sample.cs

```diff
@@ -45,12 +49,23 @@ static void Main()
                         Thread.Sleep(measurementDuration);
 
                         // Print out the measured data
-                        bme680.TryReadTemperature(out var temperature);
-                        bme680.TryReadPressure(out var pressure);
-                        bme680.TryReadHumidity(out var humidity);
-                        bme680.TryReadGasResistance(out var gasResistance);
-
-                        Console.WriteLine($"{temperature.Celsius:N2} \u00B0C | {pressure.Hectopascal:N2} hPa | {humidity:N2} %rH | {gasResistance:N2} Ohm");
+                        bme680.TryReadTemperature(out var tempValue);
+                        bme680.TryReadPressure(out var preValue);
+                        bme680.TryReadHumidity(out var humValue);
+                        bme680.TryReadGasResistance(out var gasResistance);                        
+                        var altValue = WeatherHelper.CalculateAltitude(preValue, defaultSeaLevelPressure, tempValue);
+                    
+                        Console.WriteLine($"Gas resistance: {gasResistance:0.##}Ohm");
+                        Console.WriteLine($"Temperature: {tempValue.Celsius:0.#}\u00B0C");
+                        Console.WriteLine($"Pressure: {preValue.Hectopascal:0.##}hPa");                    
+                        Console.WriteLine($"Altitude: {altValue:0.##}m");                    
+                        Console.WriteLine($"Relative humidity: {humValue:0.#}%");
+                        Console.WriteLine($"Heat index: {WeatherHelper.CalculateHeatIndex(tempValue, humValue).Celsius} \u00B0C");
+                        Console.WriteLine($"Summer simmer index: {WeatherHelper.CalculateSummerSimmerIndex(tempValue, humValue).Celsius} \u00B0C");
+                        Console.WriteLine($"Saturated vapor pressure: {WeatherHelper.CalculateSaturatedVaporPressure(tempValue).Hectopascal} hPa");

```

Comment:
similar comment about spaces before unit (here and above and below)

Reply:
Fixed all.